appdev/index.rst

```diff
@@ -173,7 +173,10 @@ A collection of external resources
 System Software
 ---------------
 
-Working on system components
+Ubuntu Touch image ship a set of applications also known as "core apps".
+See `core apps repositories on github <https://github.com/search?q=topic%3Acore-app+org%3Aubports&type=Repositories>`__
+
+This resource will help you get started with working on system components
```

Comment:
It's good that this is being clarified. However, this doesn't really hit the goal of the page it links to. The article does list how to build core apps, but it's more about building any package installed on the Ubuntu Touch system. This includes libraries, system services, and other components that are preinstalled.

Reply:
ok, i also thought maybe it was more than core apps development.
That is fine i close this PR. Will see later then where is the best place to talk about "core apps"